README.md

```diff
@@ -61,6 +61,25 @@ The release process is building all runtimes and then puts them into a release i
 
 The format of [`CHANGELOG.md`](CHANGELOG.md) is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## Release guidelines
+
+Here are the recommended steps for both major and minor runtime releases.
+
+|Steps |Description |Major |Minor |
+|------|------------|------|------|
+|0 |Open an [issue](https://github.com/polkadot-fellows/runtimes/issues) for the release in the runtimes repo with the **version number**. | ✔️ | ✔️ |
+|1 |Identify **released crates**. | ✔️ | ❌ |
+|2 |Identify and monitor **potential blockers** (old dependencies, pending or failed upgrades). | ✔️ | ✔️ |
+|3 |Identify and include **PRs** with required tests, highlighting the integration tests that have changed. | ✔️ | ✔️ |
+|4 |Identify and communicate all details about **potential breaking changes** (transaction/event/error encoding, polkadot-sdk migrations, XCM and storage format, etc.) or **disruptions**. Make sure to **ping @SBalaguer and @anaelleltd** in your commentary. | ✔️ | ✔️ |
+|5 |Run **benchmarking** for changed pallets. | ✔️ | ✔️ |
```

Comment:
It would be nice if these steps were links to some doc on how you actually do them. Below we already have a link to the opengov-cli tool, that's good, but to my knowledge it is a bit more involved in practice.

Reply:
Good point.

These guidelines only aim to give an overview of the key stages of runtime releases, so that Fellowship members and the wider community can have a basic understanding of what goes on behind the scenes and identify how/where they can contribute.

Given that this is collaborative and dynamic process through and through, it would be hard to create detailed tutorials with specific troubleshooting tips for each step. 

Still, I will do my best to source relevant docs and link them here for reference. Thanks for the suggestion!

Reply:
https://github.com/polkadot-fellows/runtimes/blob/main/docs/weight-generation.md

Reply:
> https://github.com/polkadot-fellows/runtimes/blob/main/docs/weight-generation.md

Thank you! I will be adding this link to step 5.